suitcase/pizza-box/handlers.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+import humanize
+import pandas as pd
+import os
+from databroker.tests.utils import temp_config
+from databroker import Broker
+from databroker.assets.handlers_base import HandlerBase
+
+# this will create a temporary databroker object with nothing in it
+db = Broker.from_config(temp_config())
+
+fc = 7.62939453125e-05
+adc2counts = lambda x: ((int(x, 16) >> 8) - 0x40000) * fc \
+        if (int(x, 16) >> 8) > 0x1FFFF else (int(x, 16) >> 8)*fc
+enc2counts = lambda x: int(x) if int(x) <= 0 else -(int(x) ^ 0xffffff - 1)
```

Comment:
Some brief explanation of the operations above would not hurt, so a new developer could easier understand the purpose.